### ISSUE TITLE
some confusion about the code

### ISSUE THREAD

**Issue by stanpcf** [lephong/mulrel-nel#12]
Hi, thx for your great work about nel. I have reviewed your code, and have some questions. 

## 1. how to generate dict and embedding in dir `data/generated/embeddings/word_ent_embs/glove`
I find two word_embedding.npy which is used in your paper.

the large embedding which is in the `data/generated/embeddings/word_ent_embs\word_embeddings.npy` used to compute the local-ctx-attention-ranker which is from G&H's EMNLP paper and the candidate select. the vocab size is **492408**

the small embedding which is in the `data/generated/embeddings/word_ent_embs/glove\word_embeddings.npy` used to compute _f(mi, ci)_ which is showed in Figure3 in your paper and the vocab is **60862**

the large embedding may be the pretrained glove or word2vec vector. But I don't know how to generate the small embedding(which size is 60862) and why you don't use the large embedding to representation the word in the _f(mi, ci)_

## 2. the ntee model
In your paper, the ntee model is used for prerank the candidate entities, but you just use matrix-matrix product the mean(context-word-representation) and candidate entities. the original ntee model is poor in your work?

thx for your great work

**Comment by lephong** [comment]
1. Re the second embedding, I tried both word2vec and glove and found glove better. From the full glove embeddings (http://nlp.stanford.edu/data/glove.840B.300d.zip), I selected only those entries that appear at least 2 (or 3) times in the training set, and all entries that appear in the test/dev set. 

2. Re NTEE model, it's a legacy. In the beginning, I implemented https://github.com/studio-ousia/ntee , but couldn't make it work.
